src/utils/logic-app/templateUtils.ts

```diff
@@ -6,6 +6,7 @@
 import { Workflow } from "azure-arm-logic/lib/models";
 import * as jsyaml from "js-yaml";
 import { normalizeParameterName, normalizeResourceName } from "../stringUtils";
+import { pathMatch } from "tough-cookie";
```

Comment:
Add the `tough-cookie` NPM module in package.json.  Also, `tough-cookie` is BSD-3, not MIT, licensed so we need to get OSS approval.

Reply:
Not sure how it gets here actually... It's not used in this file, will remove.